lib/wprof/reporters/db_report.rb

```diff
@@ -0,0 +1,16 @@
+module Wprof
+  module Reporters
+    module DatabaseReport
+      def db_report
```

Comment:
Beware of the explicity of method

Reply:
I don't follow.

Reply:
I believe it's because this method without a `module_function` or with def `self.` is meant to be included ?

In other words, do you use DatabaseReport as an extension to another class, or as a singleton ?